src/test/java/de/tum/in/www1/artemis/domain/notification/NotificationPlaceholderSignatureTest.java

```diff
@@ -0,0 +1,80 @@
+package de.tum.in.www1.artemis.domain.notification;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+import org.junit.jupiter.api.Test;
+
+import com.google.gson.GsonBuilder;
+
+import de.tum.in.www1.artemis.AbstractSpringIntegrationIndependentTest;
+import io.github.classgraph.AnnotationEnumValue;
+import io.github.classgraph.ClassGraph;
+
+class NotificationPlaceholderSignatureTest extends AbstractSpringIntegrationIndependentTest {
+
+    /**
+     * If this test fails you have changed the notification placeholder files. This may have dramatic consequences on the mobile applications (iOS & Android) and the database.
+     * Changing the notification placeholders files changes the following things:
+     * 1. Other key value pairs are sent to the clients. This may mean that the native clients no longer understand them and notifications are no longer shown to users.
+     * 2. The database becomes inconsistent. The placeholders are stored as Json in the database.
```

Comment:
```suggestion
     * If this test fails, you have changed the notification placeholder files. This may have dramatic consequences on the mobile applications (iOS & Android) and the database.
     * Changing the notification placeholders files changes the following things:
     * 1. Other key value pairs are sent to the clients. This may mean that the native clients no longer understand them and notifications are no longer shown to users.
     * 2. The database becomes inconsistent. The placeholders are stored as JSON in the database.
```